### ISSUE TITLE
Geting records without using pihole

### ISSUE THREAD

**Issue by zonexCS** [wg-easy/wg-easy#919]
Hello, today I installed pihole with wireguard easy, and when i log on admin panel I see some activities, but the questions is, how can I block them or something? Because I didnt put the dns on any device and i got 3 diferents entries
![image](https://github.com/wg-easy/wg-easy/assets/20957928/885d0fd5-e707-46c3-aca3-d8b40973fe23)

**Comment by eandersons** [comment]
Obviously the host is already configured to use Pi-hole, and Docker containers by default are using the same DNS. From the relevant section in Docker documentation - [DNS services](https://docs.docker.com/network/#dns-services):

> Containers use the same DNS servers as the host by default [..]

As clients' traffic is routed through the WireGuard Easy container, Pi-hole receives and logs DNS requests.
If you do not want that, you can configure WireGuard Easy container to use a different DNS server.

**Comment by zonexCS** [comment]
Yes, but there shows alot of conectios trought my networks... There only have to show my devices like my pc, my phone, my TV... and as you can see there are alot of conections trought my newtork
![image](https://github.com/wg-easy/wg-easy/assets/20957928/53e8d2a5-7b32-4d08-bcb0-a580d59962a6)

**Comment by eandersons** [comment]
That could indicate that the port 53 is publicly available.

**Comment by denisszzz** [comment]
> Yes, but there shows alot of conectios trought my networks... There only have to show my devices like my pc, my phone, my TV... and as you can see there are alot of conections trought my newtork ![image](https://private-user-images.githubusercontent.com/20957928/313436035-53e8d2a5-7b32-4d08-bcb0-a580d59962a6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3MTQ1ODEsIm5iZiI6MTcxMDcxNDI4MSwicGF0aCI6Ii8yMDk1NzkyOC8zMTM0MzYwMzUtNTNlOGQyYTUtN2IzMi00ZDA4LWJjYjAtYTU4MGQ1OTk2MmE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE3VDIyMjQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlYTI5MzM3MTM4NTU1MWJiYzE2NWM4ZWZjN2QzZWVkZDNjN2Q3Y2U3Yzk3NmQ0ZmI3ZmMwZWI2YWNmZmZkMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._dx0h3YSEwg-NXkvPVgBBLNxowtNQ7N8W16ybCE7-lo)

all ur connections from wireguard vpn looks like wg-easy.wg-easy-pihole_wg-easy. Its becouse wireguard has masquerade users ip addresses.

**Comment by zonexCS** [comment]
> That could indicate that the port 53 is publicly available.

So how i check if they are using it to enter to my rasbperry? Obviusly has to be open to work... but how to make it safe?

**Comment by zonexCS** [comment]
Ill check latter this file but yesterday with my phone via SSH I didnt found this file... I install all dockers with portainer im not sure if this is an error or something... this night when I go home i'll check again

**Comment by zonexCS** [comment]
> > Yes, but there shows alot of conectios trought my networks... There only have to show my devices like my pc, my phone, my TV... and as you can see there are alot of conections trought my newtork ![image](https://private-user-images.githubusercontent.com/20957928/313436035-53e8d2a5-7b32-4d08-bcb0-a580d59962a6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA3MTQ1ODEsIm5iZiI6MTcxMDcxNDI4MSwicGF0aCI6Ii8yMDk1NzkyOC8zMTM0MzYwMzUtNTNlOGQyYTUtN2IzMi00ZDA4LWJjYjAtYTU4MGQ1OTk2MmE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE3VDIyMjQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlYTI5MzM3MTM4NTU1MWJiYzE2NWM4ZWZjN2QzZWVkZDNjN2Q3Y2U3Yzk3NmQ0ZmI3ZmMwZWI2YWNmZmZkMTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._dx0h3YSEwg-NXkvPVgBBLNxowtNQ7N8W16ybCE7-lo)
> 
> all ur connections from wireguard vpn looks like wg-easy.wg-easy-pihole_wg-easy. Its becouse wireguard has masquerade client addresses.
> 
> you can see this if you open config file /root/.wg-easy/wg0.cong 11 line `PostUp = iptables -t nat -A POSTROUTING -s 10.8.0.0/24 -o eth0 -j MASQUERADE; iptables -A INPUT -p udp -m udp --dport 51820 -j ACCEPT; iptables -A FORWARD -i wg0 -j ACCEPT; iptables -A FORWARD -o wg0 -j ACCEPT; `

Ill check latter this file but yesterday with my phone via SSH I didnt found this file... I install all dockers with portainer im not sure if this is an error or something... this night when I go home i'll check again

**Comment by eandersons** [comment]
> > That could indicate that the port 53 is publicly available.
> 
> So how i check if they are using it to enter to my rasbperry? Obviusly has to be open to work... but how to make it safe?

Either use firewall or map container port to host's local IP address (for example, `192.168.1.1:53:53` instead of `53:53` which is equivalent to `0.0.0.0:53:53` and makes the port 53 accessible to the outside world) so the port 53 would be accessible only from within your network.
From [Published ports](https://docs.docker.com/network/#published-ports) in Docker documentation:

> Publishing container ports is insecure by default. Meaning, when you publish a container's ports it becomes available not only to the Docker host, but to the outside world as well.
>
> If you include the localhost IP address (127.0.0.1) with the publish flag, only the Docker host can access the published container port.
